auction-app-backend/src/main/java/com/example/auctionapp/service/ProductService.java

```diff
@@ -25,4 +25,7 @@ Page<ProductBidDetailsResponse> getProductByUserAndStatus(final UUID userId,
                                                               final ProductStatus productStatus,
                                                               final int page,
                                                               final int size);
+
+    Boolean hasActiveProducts(final UUID userId);
```

Comment:
boolean instead of Boolean can be used 
We don't have any complex operations on the results, and boolean requires less memory